sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Add a brief note that this model is being trained on random data for the sake of the example, and does not have any meaning. Or alternatively, and more typical of notebooks, you could use an existing dataset to add some context and meaning to the example model. However for Inference Recommender, I understand that the model itself isn't the main point of the feature.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks I've added an explanation. I prefer using random data so we train a model quickly and move on to inference. Using datasets adds external dependencies and I've seen for some inference examples that it complicates the overall notebook.</p>